projects/laji/src/app/shared/api/WarehouseApi.ts

```diff
@@ -299,6 +299,18 @@ export class WarehouseApi {
     return this.http.post<any>(path, undefined, {params: queryParameters});
   }
 
+  public downloads(id: string, extraHttpRequestParams?: any): Observable<any> {
```

Comment:
A small thing, but it would be good practice to default to using the `unknown` type for API responses if we don't explicitly know the type.